wpilibc/src/main/native/cpp/logging/CSVLogCell.cpp

```diff
@@ -0,0 +1,38 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "frc/logging/CSVLogCell.h"
+
+#include <algorithm>
+#include <iostream>
+
+using namespace frc;
+
+CSVLogCell::CSVLogCell(std::string name) {
+  if (IsStringValid(name)) {
+    m_name = name;
+  }
+}
+
+std::string CSVLogCell::GetName() { return m_name; }
+
+std::string CSVLogCell::GetContent() { return m_content; }
```

Comment:
This function signature should be `const std::string& CSVLogCell::GetContent() const` since m_content isn't modified here.